src/platforms/common/configuration/options.mdx

```diff
@@ -577,13 +577,36 @@ Switches out the transport used to send events. How this works depends on the SD
 
 </ConfigKey>
 
-<ConfigKey name="http-proxy" supported={["node", "php", "python", "java", "dotnet", "rust"]}>
+<ConfigKey name="transport-options" supported={["javascript", "node"]}>
+
+Options used to configure the transport. This is an object with the following possible optional keys:
+
+<PlatformSection supported={["node"]}>
+
+- `headers`: An object containing headers to be sent with every request.
+- `proxy`: A proxy used for outbound requests. Can be http or https.
+- `caCerts`: A a path to a CA certificate, a list of paths to a CA certificate, or a buffer of CA certificates
```

Comment:
Ah much clearer. Fixing a typo, adding a period, and making a suggestion for a shorter version:
```suggestion
- `caCerts`: A path or list of paths to a CA certificate, or a buffer of CA certificates.
```